src/pipeline_config.py

```diff
@@ -210,6 +210,13 @@
                                   },
         },
     },
+    'tta_generator': {'flip_ud': True,
+                      'flip_lr': True,
+                      'rotation': True,
+                      'color_shift_runs': False},
```

Comment:
@taraspiotr do those color transforms work out of the box here? I would hope so but idk

Reply:
@jakubczakon they do work, but the results were slightly worse